### DISCUSSION TITLE
Change colors to monochrome

### DISCUSSION THREAD

**Discussion by MarvinKweyu** [MarvinKweyu/ColorDetect#16] [Ideas]
There should be a way, either now or in the future, to change the colors to monochrome. That is, the colors identified should be the only ones colored from within the image. How feasible can this be? 

For example, if I identify red in an image, the rest of the image should turn to black and white, and so on. 

Got a question? Chat right here!

New ? No problem. Have a look at the project [documentation](https://colordetect.readthedocs.io/en/master/colordetect.html). Remember to get the [contribution guidelines](https://github.com/MarvinKweyu/ColorDetect/blob/master/CONTRIBUTING.rst)

**Comment by qinyanjuidavid** [comment]
Sounds awesome, I would love to see it happen...

**Comment by MarvinKweyu** [comment]
Currently in progress on pull #28
